numpy/random/__init__.pyi

```diff
@@ -1,4 +1,8 @@
 from typing import Any, List
+from numpy.random.bit_generator import (
```

Comment:
I assume `SFC64` and `PCG64` will have to be imported as well?

Reply:
Thanks.  They have been.